DigitalLearningSolutions.Data/Models/TutorialContent/TutorialVideo.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace DigitalLearningSolutions.Data.Models.TutorialContent
+{
+    public class TutorialVideo
+    {
+        public string TutorialName { get; }
+        public string CourseTitle { get; }
+        public string? VideoPath { get; }

```

Comment:
Should this really be nullable? It won't be a valid tutorial video without a path, so we'll want to redirect to a 404 I think. In which case it might be better to get the service method to return null, if that makes sense? 🙂

Reply:
I agree, but Dapper is still willing to make a TutorialVideo with a `null` `VideoPath` even if it isn't marked as nullable. Should I check for this in the `GetTutorialVideo` method? 🙂

Reply:
Yes, I think I'd do that 🙂

Reply:
Added in 7ce3621 🙂

Reply:
Wait actually we'd still need to mark it as optional then 🙁 Could we throw an exception in the contructor? I think we did that in a similar situation? 🙂

Reply:
Oh yeah, I had forgotten about that! Done in 41b5ea6 🙂